ftp-ballerina/listener_endpoint.bal

```diff
@@ -29,7 +29,7 @@ public class Listener {
     # Gets invoked during object initialization.
     #
     # + listenerConfig - Configurations for FTP listener
-    public isolated function init(ListenerConfiguration listenerConfig) {
+    public isolated function init(*ListenerConfiguration listenerConfig) {
```

Comment:
Don't we need to add mandatory fields like `host`, `port` as the parameters of the function, instead of configurations? Just asking since there are API changes anyway. WDYT? @Maninda @shafreenAnfar